examples/compiled/histogram_log.vg.json

```diff
@@ -41,6 +40,7 @@
           "maxbins": 10,
           "extent": {"signal": "bin_maxbins_10_log_x_extent"}
         },
+        {"type": "formula", "expr": "log(datum.x)/log(10)", "as": "log_x"},
```

Comment:
Oh, this is not good.

Reply:
@domoritz I think this is because of the bug that I fixed earlier in the field intersection function, but it looks like it got dropped in the rebase. I re added the suggestion inline that fixed it before